README.md

```diff
@@ -190,6 +194,29 @@ If you just want to experiment with Jenkins X, you can set force_destroy to true
 
 :warning: **Note**: If you set `force_destroy` to false, and run a `terraform destroy`, it will fail. In that case empty the s3 buckets from the aws s3 console, and re run `terraform destroy`.
 
+### EKS node groups
+This module provisions self-managed worker nodes by default.
+In order to provision EKS node groups (EKS managed worker groups) create a _main.tf_ with the following content:
```

Comment:
what's about adding a link to some relevant documentation, for example, make "EKS node groups" a link to https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html. 

A key benefit might be useful as well. I don't think we need much here, but just mentioning one key feature might help to make a decision on whether this is something one wants to enable.
I personally a much more familiar with GKE and feel a bit overpowered by all the options and bells and whistles we are adding here.

Reply:
> what's about adding a link to some relevant documentation, for example, make "EKS node groups" a link to https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html.

:100: Will do :+1: 

> A key benefit might be useful as well. I don't think we need much here, but just mentioning one key feature might help to make a decision on whether this is something one wants to enable.

Agreed, makes sense.

> I personally a much more familiar with GKE and feel a bit overpowered by all the options and bells and whistles we are adding here.

It's just all the complexities and options that aws provides ... I have very limited knowledge of GCP though ... We need really good documentation, else it will become pretty hard for someone trying it. AWS is confusing imo ...